python/ray/tune/tune.py

```diff
@@ -59,7 +78,7 @@ def run(run_or_experiment,
         scheduler=None,
         with_server=False,
         server_port=TuneServer.DEFAULT_PORT,
-        verbose=2,
+        verbose=1,
```

Comment:
We should probably keep the previous behavior here and feature-flag this output.